### PR TITLE
[RLC] Remove typedoc and disable docs generation

### PR DESCRIPTION
**Pull request by joheredi**
Remove typedoc and disable docs generation

### CODE REVIEW

File: src/generators/static/packageFileGenerator.ts [hunk 1]
```diff
@@ -134,8 +134,7 @@ function restLevelPackage(packageDetails: PackageDetails) {
       "unit-test": "echo skipped",
       "unit-test:node": "echo skipped",
       "unit-test:browser": "echo skipped",
-      docs:
-        "typedoc --excludePrivate --excludeExternals  --out ./dist/docs ./src"
+      docs: "echo skip"
```
Comment (deyaaeldeen) [inline src/generators/static/packageFileGenerator.ts:137]: Is there a need to keep the script?

**Review by deyaaeldeen — APPROVED** [review]

File: src/generators/static/packageFileGenerator.ts (hunk 1 above)
Comment (joheredi) [reply src/generators/static/packageFileGenerator.ts:137]: We can remove it. Do you know if rush would complain? If not I'll get rid of it

Comment (deyaaeldeen) [reply src/generators/static/packageFileGenerator.ts:137]: I don't believe rush will complain about the lack of the docs script. I don't have it in my libraries and I didn't see any complaints regarding it. Also for context, we used to have that script but we decided to remove it at some point a few months ago.

Comment (joheredi) [reply src/generators/static/packageFileGenerator.ts:137]: I'll remove it

**Review by qiaozha — APPROVED** [review]

**Pull request merged**